### PR TITLE
LPS-161995 [EVP] Create Master Page Layout

### PR DESCRIPTION
**Pull request by juliocezarjc**
[LPS-161995](https://issues.liferay.com/browse/LPS-161995)

### CODE REVIEW

**Comment by juliocezarjc** [comment]
ci:test:sf

**Comment by juliocezarjc** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/clay-icon-link/configuration.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+				{
+					"dataType": "string",
+					"defaultValue": "check-circle",
+					"label": "icon-name",
+					"name": "iconName",
+					"type": "text"
+				},
+				{
+					"dataType": "string",
+					"defaultValue": "",
```
Comment (DiogoNo) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/clay-icon-link/configuration.json:14]: Acho que nao precisa colocar um campo sem ter value

```suggestion
				
```

**Comment by juliocezarjc** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/clay-icon-link/configuration.json (hunk 1 above)
Comment (juliocezarjc) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/clay-icon-link/configuration.json]: Adjusted.

**Comment by juliocezarjc** [comment]
ci:test:sf

**Comment by juliocezarjc** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by juliocezarjc** [comment]
ci:test:sf

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @juliocezarjc

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by juliocezarjc** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**